oc_cdtapi/DmsGetverAPI.py

```diff
@@ -46,6 +93,92 @@ def create_distr_request(self, version=None, source_version=None, distr_type=Non
 
         return distr_state_info
 
+    def download_file(self, image_id):
+        """
+        Downloads specified image to file
+        :param str image_id: id of image to download
+        :return: path to downloaded file
+        """
+        logging.debug('Reached download_file')
+        logging.debug('image_id: [%s]' % image_id)
+        logging.debug('Unsetting auth')
+        self.web.auth = None
+        headers = self.get_headers()
+        url = posixpath.join('api', 'v1', 'images', image_id, 'download')
+        logging.debug('url: [%s]' % url)
+        tf = tempfile.NamedTemporaryFile(delete=False)
+        tfn = tf.name
+        logging.debug('tf: [%s]' % tfn)
+        r = self.get(url, headers=headers, stream=True)
+        for chunk in r.iter_content(chunk_size=8192):
+            tf.write(chunk)
+        tf.seek(0)
+        return tfn
```

Comment:
Useless method since temporary file will be destroyed immediately after download, and name only is returned.
Data will be lost.

Reply:
note (delete=False) in temp file creation